packages/react-component-library/src/components/TopLevelNavigation/Masthead/Masthead.test.tsx

```diff
@@ -217,13 +217,33 @@ describe('Masthead', () => {
       expect(wrapper.queryByTestId('not-read')).toBeInTheDocument()
     })
 
+    it('should set the nav `role` to `menubar`', () => {
+      expect(wrapper.getByTestId('masthead-nav-list')).toHaveAttribute(
+        'role',
+        'menubar'
+      )
+    })
+
+    it('should set each nav list item `role` to `none`', () => {
+      const listItems = wrapper.getAllByTestId('masthead-nav-item')
+
+      listItems.forEach((listItem) => {
+        expect(listItem).toHaveAttribute('role', 'none')
```

Comment:
What's the significance of a role of `none`?

Reply:
Taken from https://www.w3.org/TR/wai-aria-practices-1.1/examples/menubar/menubar-1/menubar-1.html

```
- Removes the implied listitem role of the li element.
- Necessary because the parent ul is serving as a menu so the li elements are not in their required list context.
```